src/react-apps/applications/runtime/src/features/form/workflow/reducer/index.ts

```diff
@@ -44,11 +47,14 @@ const WorkflowReducer: Reducer<IWorkflowState> = (
         $set: {
           state: state.state,
           error,
-        }
+        },
       });
     }
     case ActionTypes.SET_CURRENT_STATE: {
       const workflowState = (action as ISetCurrentState).state;
+      if (workflowState === WorkflowSteps.Archived) {
+        document.body.className = 'a-bgGreenLight flex-column d-flex';
+      }
```

Comment:
This should be done in the saga, not in the reducer.
The reducer should ONLY update the redux-state.

Reply:
Fixed